packages/components/src/form-token-field/index.js

```diff
@@ -193,38 +201,14 @@ class FormTokenField extends Component {
 		const separator = this.props.tokenizeOnSpace ? /[ ,\t]+/ : /[,\t]+/;
 		const items = text.split( separator );
 		const tokenValue = last( items ) || '';
-		const inputHasMinimumChars = tokenValue.trim().length > 1;
-		const matchingSuggestions = this.getMatchingSuggestions( tokenValue );
-		const hasVisibleSuggestions = inputHasMinimumChars && !! matchingSuggestions.length;
 
 		if ( items.length > 1 ) {
 			this.addNewTokens( items.slice( 0, -1 ) );
 		}
 
-		this.setState( {
-			incompleteTokenValue: tokenValue,
-			selectedSuggestionIndex: -1,
-			selectedSuggestionScroll: false,
-			isExpanded: false,
-		} );
+		this.setState( { incompleteTokenValue: tokenValue }, this.updateSuggestions );
```

Comment:
Technically we could avoid `this.updateSuggestions` as the callback here and consolidate instead to `componentDidUpdate` (which can also detect the state change).